File: Tech Module/Software Technologies/19. Java Basics - Lab/src/Three_Integers_Sum.java
import java.util.Locale;
import java.util.Scanner;

public class Three_Integers_Sum {
    public static void main(String[] args) {
        Locale.setDefault(Locale.ROOT);
        Scanner scanner = new Scanner(System.in);

        String input = scanner.nextLine();
        String[] stringArray = input.split(" ");
        int[] arr = new int[stringArray.length];
        for(int i = 0; i < stringArray.length; i++){
            arr[i] = Integer.parseInt(stringArray[i]);
        }
        CheckIfExist(arr[0], arr[1], arr[2]);
    }

    public static void CheckIfExist(int firstNum, int secondNum, int thirdNum){

        if(firstNum == secondNum + thirdNum){
            int min = secondNum < thirdNum ? secondNum : thirdNum;
            int max = secondNum > thirdNum ? secondNum : thirdNum;
            System.out.println("" + min + " + " + max + " = " + firstNum);
        }else if(secondNum == firstNum + thirdNum){
            int min = firstNum < thirdNum ? firstNum : thirdNum;
            int max = firstNum > thirdNum ? firstNum : thirdNum;
            System.out.println("" + min + " + " + max + " = " + secondNum);
        }else if(thirdNum == secondNum + firstNum){
            int min = firstNum < secondNum ? firstNum : secondNum;
            int max = firstNum > secondNum ? firstNum : secondNum;
            System.out.println("" + min + " + " + max + " = " + thirdNum);
        }else {
            System.out.println("No");
        }
    }

}

File: 02. Tech Module/Software Technologies/7. JavaScript Syntax and Basic Web/JavaScript - Syntax and Basic Web/ThreeIntegersSum.js
function solution([nums]) {
    let array = nums.split(' ');
    let a = Number(array[0]);
    let b = Number(array[1]);
    let c = Number(array[2]);

    if (a + b === c) {
        console.log(`${Math.min(a,b)} + ${Math.max(a,b)} = ${c}`);
    }else if (a + c === b) {
        console.log(`${Math.min(a,c)} + ${Math.max(a,c)} = ${b}`);
    }else if (b + c === a) {
        console.log(`${Math.min(c,b)} + ${Math.max(c,b)} = ${a}`);
    }else{
        console.log('No');
    }
}

solution('3 8 12');